Add/update diversity and inclusion policies (#83)
confidential-computing/governance: diversity-and-inclusion-policies.md

# Diversity and Inclusion Policy Guidelines

## Intro
The Confidential Computing Consortium is a community focused on open source
licensed projects securing DATA IN USE & accelerating the adoption of
confidential computing through open collaboration.

Every member is welcome; every project meeting our criteria is welcome. 
We are a transparent, collaborative community.

We as members, contributors, and leaders pledge to make participation
in our community a harassment-free experience for everyone.

## Policies
The [Project Proposal Template](project-submission-template.md) specifies that

> a project must have a Code of Conduct to be submitted to the Confidential

Reviewer: Technically speaking this is not strictly true.   Someone could submit and answer "none".  The TAC would then review and decide whether to accept it or not.  The TAC could choose to ask for one before accepting it, but someone can submit a project before then.
```suggestion
a project should have a Code of Conduct to be accepted by the Confidential
```

Reviewer: TAC consensus is in support of Dan's proposed wording

Author: TAC consensus for "submitted"

Computing Consortium.

The [Project Progression Policy](project-submission-template.md) specifies

> additional community characteristics that a project must demonstrate for each

Reviewer: ```suggestion
additional community characteristics that a project should demonstrate for each
```
